netbout-web/src/test/java/com/netbout/rest/bout/TkIndexTest.java

```diff
@@ -64,10 +64,12 @@ public void rendersBoutPage() throws Exception {
         MatcherAssert.assertThat(
             XhtmlMatchers.xhtml(
                 new RsPrint(
+                    // @checkstyle MultipleStringLiteralsCheck (1 line)
                     new FkBout(".*", new TkIndex(base)).route(
```

Comment:
@erimerturk let's extract this shares string into a private static final variable since it pertains to multiple test methods.